src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -2867,3 +2867,29 @@ def test_python_unhandled_exception(self):
     self._validate_get_crash_data(data, expected_type, expected_address,
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
+                                  
+  def test_gdb_sigtrap(self):
+    """Test for GDB stack."""
+    data = self._read_test_data(
+        'gdb_sigtrap.txt')
+    expected_type = 'SIGTRAP'
+    expected_address = '0x000000000ac8'
+    expected_state = ('__aeabi_memcpy\nxymodem_trnasfer\nLoadImageFromUsb30\n')
+    expected_stacktrace = data
+    expected_security_flag = True
+    self._validate_get_crash_data(data, expected_type, expected_address,
+                                  expected_state, expected_stacktrace,
+                                  expected_security_flag)
+                                  
+  def test_gdb_sigtrap_and_libfuzzer(self):
+    """Test for GDB stack with libfuzzer."""
+    data = self._read_test_data(
+        'gdb_sigtrap_and_libfuzzer.txt')
+    expected_type = 'SIGTRAP'
+    expected_address = '0x000000000ac8'
```

Comment:
In one test/stacktrace, increase the address to a high one like 
0x007000000ac8,
then security flag should be True as expected. That way we test for both cases.

Reply:
Maybe this requires this
https://github.com/google/clusterfuzz/blob/master/src/python/crash_analysis/crash_analyzer.py#L360
instead of return true,
int_address = address_to_integer(crash_address)
return is_null_dereference(int_address)

Reply:
This is tough because the addresses for PC are all very low so it will  set the security flag to false incorrectly

Reply:
We will fix this in a separate cl